tests/test_backends_clifford.py

```diff
@@ -46,7 +46,9 @@ def test_global_backend(backend):
     set_backend(backend.name, platform=backend.platform)
     clifford_bkd = CliffordBackend()
     target = (
-        GlobalBackend().name if backend.name == "numpy" else GlobalBackend().platform
+        _Global.get_backend().name
+        if backend.name == "numpy"
+        else _Global.get_backend().platform
```

Comment:
```suggestion
        _Global._backend.name
        if backend.name == "numpy"
        else _Global._backend.platform
```

If you want to test what it is, without triggering the construction, it is acceptable to access private members (within tests is acceptable in general)

Reply:
Same as #U1.